README.md

```diff
@@ -62,11 +62,11 @@ Use of these components assumes the following from the peer project:
 - The GDS Transport font face is included ([for gov.uk domains only](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk))
 - Other than the reset, no other styles affecting generic elements (without classes, IDs etc) are present in the CSS.
 
-## Why Glamorous/CSS-in-JS?
+## Why Emotion/CSS-in-JS?
 
 See [A Unified Styling Language](https://medium.com/seek-blog/a-unified-styling-language-d0c208de2660).
 
-We have opted for [Glamorous](https://glamorous.rocks/) over styled-components as we like the JS syntax and would rather not burden either the front end or the transpiler with a CSS parser. However, there is [an open discussion](https://github.com/penx/govuk-react/issues/219) on whether to use styled-components or Emotion, so this is subject to change.
+We have opted for [Emotion](https://emotion.sh/) over styled-components as we like the [support for JS syntax](https://emotion.sh/docs/object-styles) and would rather not burden either the front end or the transpiler with a CSS parser.
```

Comment:
"and would rather not burden either the front end or the transpiler with a CSS parser" I think can be removed now as I don't think it's really an issue